samples/function_chaining_custom_status/DurableActivity/__init__.py

```diff
@@ -1,27 +1,27 @@
-import logging
-
-def main(key: str) -> str:
-    """Activity function performing a specific step in the chain
-    
-    Parameters
-    ----------
-    key : str
-        key to a dictionary that advances the chain
-    
-    Returns
-    -------
-    [int]
-        value in the dictionary as a result
-    """
-
-    logging.warning(f"Activity Triggered: {key}")
-    switch_statement = {}
-    switch_statement["One"] = "Two"
-    switch_statement["Two"] = "Three"
-    try:
-        if switch_statement[key]:
-            return f'{switch_statement[key]}'
-    except KeyError as e:
-        return f'One'
-
-    
+import logging
+
+def main(key: str) -> str:
+    """Activity function performing a specific step in the chain
+    
+    Parameters
+    ----------
+    key : str
+        key to a dictionary that advances the chain
+    
+    Returns
+    -------
+    [int]
+        value in the dictionary as a result
+    """
+
+    logging.warning(f"Activity Triggered: {key}")
+    switch_statement = {}
+    switch_statement["One"] = "Two"
```

Comment:
Is there a reason we don't simplify and just have this return integers and just have it return `input+1`?

Reply:
No reason, I inherited this sample so I had not reviewed it too much. Will simplify

Reply:
done!